file

```diff
@@ -0,0 +1 @@
+This is an empty file
```

Comment:
Don't you mind to set a proper name and content to file?

Something with some sense, please, not a _file_ as filename and without any type of content.

Reply:
I've change the filename.